boards/arm/sam_v71_xult/sam_v71_xult_defconfig

```diff
@@ -14,6 +14,7 @@ CONFIG_USART_SAM=y
 CONFIG_BOARD_SAM_V71_XULT=y
 CONFIG_WDT_DISABLE_AT_BOOT=y
 CONFIG_BUILD_OUTPUT_HEX=y
+CONFIG_GPIO=y
```

Comment:
Why is this necessary?

Reply:
I was discussing with @iperry offline. Most boards have CONFIG_GPIO=y, so samples like samples/drivers/can don't even bother to set CONFIG_GPIO=y, even though blinky does. This is causing a twister error.

And the board porting guidelines (https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#general-recommendations) do say that upstream boards should enable any drivers needed for LEDs, buttons, etc., which this board doesn't seem to be doing. So it seemed OK to add it here.

Reply:
> samples/drivers/can fails to compile because the CAN example blinks an LED.

This means that CAN should select GPIO at prj.conf instead at board level. This is a sample config error and not a missing board configuration. At board level only configs that are strictly necessary to put board to run should be enabled, otherwise firmware can be bigger than expected.

Reply:
@nandojve you've marked this as resolved but you still have a -1 -- is there something else you want to see?

Reply:
I removed the first commit, squashed the remaining three, and rebased against main.

Thanks for the patience!